packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -211,14 +212,16 @@ export default class DropdownCore extends React.Component<DropdownCoreProps> {
 
         return (
             <View
+                ref={(node) =>
+                    (this.node = ((ReactDOM.findDOMNode(node): any): Element))
```

Comment:
Maybe rename `this.node` to `this.element` since it's being cast as a `Element`.